site/app/entities/poll/Poll.php

```diff
@@ -27,6 +28,14 @@ class Poll {
 
     #[ORM\Column(type: Types::TEXT)]
     protected $status;
+    /**
+     * @var string
+     */
+    #[ORM\Column(type: Types::STRING)]
+    protected $duration;
```

Comment:
Is there a reason why this is a PHPDoc annotation?  It would be better to use a native PHP type annotation if possible.
```suggestion
    #[ORM\Column(type: Types::STRING)]
    protected string $duration;
```

Reply:
Fixed